documentation/docs/98-configuration.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Configuration
+---
+
+Your project's configuration lives in a `svelte.config.cjs` file. All values are optional. The complete list of options, with defaults, is shown here:
+
+```js
+/** @type {import('@sveltejs/kit').Config} */
+module.exports = {
+	// options passed to svelte.compile (https://svelte.dev/docs#svelte_compile)
+	compilerOptions: null,
```

Comment:
```suggestion
	compilerOptions: {},
```

Maybe? To make it clear this is an object.

Although maybe not, if null is the real default.

Reply:
it actually is the default, yeah. i thought about adding the compiler options from Svelte but it felt like a maintenance burden so i didn't

Reply:
No, I think leaving the options over in the Svelte docs instead of here is the right call. I was just wondering whether specifying an empty object here made it more obvious that, if you pass something in, it should be an object. I don't feel especially strongly about this.